### ISSUE TITLE
Make new thread for file transfer

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#155]
```
If file transfer doesn't work, the application hang.
Todo : make a new thread for each file transfer demand (and auto kill them on 
error or long period) 
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 27 Apr 2011 at 10:54

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 1 Sep 2011 at 11:48
- Added labels: **Priority-Medium**
- Removed labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 3 Nov 2011 at 10:50
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed and published with 3.3
```

Original comment by `Florent....@gmail.com` on 11 Nov 2011 at 9:47
- Changed state: **Deployed**
